Source/Particles/WarpXParticleContainer.H

```diff
@@ -130,6 +130,8 @@ public:
 
     virtual void InitData () = 0;
 
+    virtual void InitIonizationModule () {}
```

Comment:
This member is now called on a `WarpXParticleContainer` (not `PhysicalParticleContainer`) inside the initialization of the multi-physics modules. Therefore, it needs to be a virtual member function of `WarpXParticleContainer`.